### DISCUSSION TITLE
modal mode

### DISCUSSION THREAD

**Discussion by beixiangsiyu** [fengyuanchen/viewerjs#570] [General]
how can i show the picture imediately without click the picture when modal mode?

**Comment by fengyuanchen** [comment]
Just call `viewer.show()`, see example: [dynamic-viewer](https://fengyuanchen.github.io/viewerjs/examples/dynamic-viewer.html) ([code](https://github.com/fengyuanchen/viewerjs/blob/main/docs/examples/dynamic-viewer.html)).
